travis/test_server.py

```diff
@@ -463,6 +477,22 @@ def main(argv=None):
         return 1
     elif counted_errors != expected_errors:
         return 1
+    # no test error, let's generate .pot and msgmerge all .po files
+    must_run_makepot = (
+        os.environ.get('MQT_DISABLE_MAKEPOT') != '1' and
+        os.environ.get('TRAVIS_PULL_REQUEST') == 'false' and
+        odoo_full == 'odoo/odoo' and
+        os.environ.get('GITHUB_USER') and
+        os.environ.get('GITHUB_TOKEN')
+    )
+    if must_run_makepot:
+        with setenv('PYTHONPATH', server_path):
```

Comment:
Setting `PYTHONPATH` here is necessary so that `import odoo` or `import openerp` works (a prerequisite of `click-odoo-makepot`).